modules/network-api/src/main/java/org/apache/ignite/network/NetworkMessage.java

```diff
@@ -24,7 +24,7 @@
  */
 public interface NetworkMessage {
     /** Size of the message type (in bytes), used during (de-)serialization. */
-    static final int MSG_TYPE_SIZE_BYTES = 4;
+    int MSG_TYPE_SIZE_BYTES = 4;
```

Comment:
Did we implement a varlen type compression? Absolute majority of messages only require 2 bytes

Reply:
At this moment no, there is no varlen compression in direct marshaller

Reply:
I meant message type specifically, it has its own code somewhere, as far as I know

Reply:
No, just two shorts, no varlen yet